src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -24,7 +29,9 @@
         Product macBook = new Product("MacBook Air 2020", BigDecimal.valueOf(1399));
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
-        // TODO: add log message about successful fetched data from DB
+
+        logger.info("successful fetched data from DB");
```

Comment:
The log message 'successful fetched data from DB' is misleading since the data is not actually fetched from a database. Consider rephrasing it to reflect the use of dummy data, such as: `logger.info("Fetched dummy data for userId: {}", userId);` This would provide more accurate information in the logs.

Reply:
Changed